README.md

```diff
@@ -71,6 +71,10 @@ Built-in authorization modules can be found under CimpressTranslationsWebpackPlu
 
 The module uses the Client Credentials Grant authentication flow to obtain a valid access token. An encrypted client secret is stored in your webpack configuration and is decrypted using AWS KMS.  To configure KMS import `aws-sdk` and set the aws configuration directly. The `aws-sdk` configuration will be reused in this library.
 
+##### clientIdAuthorizer(clientId, clientSecret)
+
+The module uses the Client Credentials Grant authentication flow to obtain a valid access token. It can be used if you have an unencrypted client secret available.
+
```

Comment:
Let's make the description more distinctive from the KMS authorizer. Alternatively, I think it's a good idea to promote this authorizer: move it to first place, then modify the description of the KMS authorizer to make it stand out that it uses KMS (in the first sentence, probably).

Reply:
Did the proposed changes + added example usage of the plain authorizer.